package.json

```diff
@@ -82,6 +81,7 @@
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "babel-runtime": "^6.26.0",
     "blurhash": "^1.1.3",
+    "bufferutil": "^4.0.3",
```

Comment:
Do you need this dep?

Reply:
https://www.npmjs.com/package/ws#opt-in-for-performance-and-spec-compliance

Reply:
But shouldn't it be added to `optionalDependencies`?

Reply:
I guess we could move that here, but I'm not sure what the benefit would be, it might be better to make sure everyone has the same experience here.

Reply:
Since bufferutil and utf-8-validate need to be built, it is troublesome for users who do not have a build environment. It helps users to be able to ignore these installation failures.

Reply:
cWS already needed building on some architectures, plus we require a lot of Ruby gems with native extensions to be built.
Anyway, I added a commit to move `bufferutil` and `utf-8-validate` to optional dependencies.